### ISSUE TITLE
preferredWidgetSize not working

### ISSUE THREAD

**Issue by cgerrits** [insolite-dev/hidable#23]
As you can see in the image, I am setting height to 86.5 with preferredWidgetSize property but it just stays at the default 56.
![image](https://github.com/insolite-dev/hidable/assets/17080854/d128588c-c6ee-42dc-82c4-6d9f89ec12a6)

**Comment by theiskaa** [comment]
Interesting, thanks for creating issue @cgerrits
Gonna check it out.

**Comment by cgerrits** [comment]
@theiskaa Any update?

**Comment by theiskaa** [comment]
@cgerrits sorry, I forgot to tell you that we already fixed that on develop, but due to some other issue, we decided to not publish a new version till we fix the hiding, unhiding issue from controller.

Here you can see the details: https://github.com/insolite-dev/hidable/issues/21

**Comment by cgerrits** [comment]
@theiskaa Ah good to know, thanks for the response!

**Comment by theiskaa** [comment]
till then, we kindly ask you to use **develop** branch to solve your problem.
When we done, and publish new version we'll remind you again.

```pubspec
  hidable:
    git:
      url: https://github.com/insolite-dev/hidable
      ref: develop
```

**Comment by theiskaa** [comment]
And in addition, you can join our discord server which is our teams main communication tool. You can ask questions, see the the progress and etc.

https://insolite.io/discord

**Comment by theiskaa** [comment]
@cgerrits hi there. We've published new(v1.0.5) version of hidable, so now the problem is solved generally.
If you find any relevant issue again or do not agree with us, please comment out your opinion and re-open the issue.

Thanks for your time.
